pkg/loadtest/client_kvstore_test.go

```diff
@@ -0,0 +1,66 @@
+package loadtest_test
+
+import (
+	"encoding/binary"
+	"testing"
+
+	"github.com/interchainio/tm-load-test/pkg/loadtest"
+)
+
+func TestKVStoreClientFactoryConfigValidation(t *testing.T) {
+	testCases := []struct {
+		config loadtest.Config
+		err    bool
+	}{
+		{loadtest.Config{Size: 30}, true},  // tx size is too small
+		{loadtest.Config{Size: 40}, false}, // tx size is just right
+		{loadtest.Config{Size: 100}, false},
+		{loadtest.Config{Size: 250}, false},
```

Comment:
As a side note, in some experiments I plan to increase the tx size up to even 10KB.

Reply:
Will add some much larger ones in! 👍